editor/uiclasses.js

```diff
@@ -288,18 +304,21 @@ class FunctionNode extends Node {
             }
         }
 
-        console.log(functionsToFindZerosOf);
-
         let zeros = [];
         for (const func of functionsToFindZerosOf) {
-            zeros = zeros.concat(getZeros(func));
+            try {
```

Comment:
omg yo try block that is so epic
I love using try blocks to make the errors go away 👍👍👍